src/main.c

```diff
@@ -19,49 +19,16 @@
 #include "cx.h"
 #include <stdbool.h>
 #include "os_io_seproxyhal.h"
+#include "io.h"
 #include "ui.h"
 #include "neo.h"
+#ifdef HAVE_BAGL
 #include "bagl.h"
-
+#endif
 #define MAX_EXIT_TIMER 4098
 
 #define EXIT_TIMER_REFRESH_INTERVAL 512
 
-static void Timer_UpdateDescription() {
-    snprintf(timer_desc, MAX_TIMER_TEXT_WIDTH, "%d", exit_timer / EXIT_TIMER_REFRESH_INTERVAL);
-}
-
-static void Timer_UpdateDisplay() {
-    if ((exit_timer % EXIT_TIMER_REFRESH_INTERVAL) == (EXIT_TIMER_REFRESH_INTERVAL / 2)) {
-        UX_REDISPLAY();
-    }
-}
-
-static void Timer_Tick() {
-    if (exit_timer > 0) {
-        exit_timer--;
-        Timer_UpdateDescription();
-    }
-}
-
-static void Timer_Set() {
-    exit_timer = MAX_EXIT_TIMER;
-    Timer_UpdateDescription();
-}
-
-static void Timer_Restart() {
-    if (exit_timer != MAX_EXIT_TIMER) {
-        Timer_Set();
-    }
-}
-
-static bool Timer_Expired() {
-    return exit_timer <= 0;
-}
```

Comment:
What was the purpose of this timer feature? Are we loosing something here?

Reply:
It looks like a way to timeout UI flows. But as we don't have this on other apps, this is probably not really necessary.